contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1029,7 +1029,7 @@ select
   , null::varchar(12) as net_address
   , null::varchar(12) as net_library
   , a.usename as loginname
-  , null::bytea as context_info
+  , t.context_info::bytea as context_info
```

Comment:
upgrade script is missing for this object change. 
Without it, new definition will not take effect after minor version upgrade 
Please add new definition in 3.1.0 -> 3.2.0 upgrade script